web/src/app/admin/AdminAPIKeys.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState } from 'react'
+import makeStyles from '@mui/styles/makeStyles'
+import {
+  Button,
+  Grid,
+  Typography,
+  Card,
+  ButtonBase,
+  CardHeader,
+} from '@mui/material'
+import { Add } from '@mui/icons-material'
+import AdminAPIKeysDrawer from './admin-api-keys/AdminAPIKeysDrawer'
+import { GQLAPIKey, CreatedGQLAPIKey } from '../../schema'
+import { Time } from '../util/Time'
+import { gql, useQuery } from '@apollo/client'
```

Comment:
We're moving away from apollo in favor of urql #3240 -- the APIs are almost identical, so it shouln't be to bad to update these

Reply:
Noted and this has been addressed.